eclcli/dns/v2/record.py

```diff
@@ -189,12 +189,30 @@ def get_parser(self, prog_name):
             metavar="<zone_id>",
             help="ID of the zone which recordset you want to list belong to.",
         )
+        parser.add_argument(
+            "--limit",
+            metavar="<limit>",
+            type=int,
```

Comment:
After modifying validation function, You can put it here.
```suggestion
            type=validate_int_range,
```